rust/test_service_elections/src/lib.rs

```diff
@@ -19,7 +19,7 @@ mod service {
     const MIN_ELECTION_TIME_SECONDS: TimePointSec = TimePointSec { seconds: 60 * 60 };
 
     #[table(name = "ElectionsTable", index = 0)]
-    #[derive(Debug, Reflect, Serialize, Deserialize)]
+    #[derive(Debug, Fracpack, Reflect, Serialize, Deserialize)]
```

Comment:
Any reason why not to automagically derive Fracpack in the `table` macros?

Reply:
It seems easier to document A requires B (common in Rust), than to document A provides B. The second leads to questions like why doesn't it also provide C, D, E, ...